web/src/dialog_widget.ts

```diff
@@ -152,6 +155,32 @@ export function launch(conf: WidgetConfig): void {
 
     const $submit_button = $dialog.find(".dialog_submit_button");
 
+    if (conf.update_submit_on_state_change) {
+        const original_values: {[key: string]: unknown} = {};
+        const current_values: {[key: string]: unknown} = {};
+
+        $submit_button.prop("disabled", true);
+        $(".modal__content :input").each(function () {
```

Comment:
Can we fix this selector to apply just to modal__content inside this dialog widget? In theory we can't have other modals open, but I don't like hardcoding that in here -- it's the kind of thing that breaks if we change our assumptions in the future.